server/http.go

```diff
@@ -72,6 +73,21 @@ func (p *Plugin) connectUserToZoom(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(state) == 0 {
```

Comment:
Checking if the state is empty might catch unwanted cases. Should the OAuth state be stored as part of the last wizard step?

Reply:
@hanzei This is a sort of fix for the case where the user clicks on a connect link without doing one of the following:

- Run `/zoom connect`
- Run `/zoom start`
- Click the channel header button to start a meeting

When one of these events occurs, the KV store is primed with info about if the user has the intention of starting a meeting after connection is complete, including what channel to send the "connected" ephemeral message to after connection is complete.

This block makes it so the user can just click the link without this priming. Then we default to the bot DM channel as the "follow up" channel. Basically, if you were to do this flow with current master, connecting wouldn't work due to lack of priming:

- `/zoom connect`
- Click connect link
- Do connection
- `/zoom disconnect`
- Click already existing connect link

So this new block makes it so you can connect with just the connect link.

Reply:
Makes sense. Thanks for the explanation :+1: